app/models/user.rb

```diff
@@ -1,5 +1,6 @@
 class User < ActiveRecord::Base
   has_many :rides
+  has_many :passengers
```

Comment:
Why does a user need many passengers? If this relationship makes sense there might be a naming change that could make this more clear.